docs/_static/custom.css

```diff
@@ -152,6 +163,37 @@ a.reference-internal
     color: '#FFFFFF'
   }
 
+/* h2
```

Comment:
There are blocks of unused code, should they be added if they aren't used?

Reply:
They're not so much added as left over. The template is still stabilizing, and there has been a lot of tinkering to get this far. 

I do not wish to make the great to be the enemy of the good. The code is functional, and my task was to clear the underbrush, not to manicure the lawn.

Reply:
Also, take a look at my troubles in TP20-1, and know that I am working towards unifying presentation for both Zenko and S3C. Right now, if I fix Zenko's css file, it means I have to fix eight S3C css files to maintain parity. Assuming RelEng pulls my giblets from the fire on that end, this will become a much more tractable problem. In the meantime...